ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/rules/views/HiveAggregateIncrementalRewritingRule.java

```diff
@@ -117,17 +116,17 @@ public void onMatch(RelOptRuleCall call) {
       projExprs.add(rightRef);
       joinConjs.add(rexBuilder.makeCall(SqlStdOperatorTable.EQUALS,
```

Comment:
Instead of doing the transformation in the rewrite to AST method, let's do it here. That will be more consistent and decrease rewriting at the AST level.

In particular, this should be `SqlStdOperatorTable .IS_NOT_DISTINCT_FROM`.